e2e-tests/pitr/run

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+write_initial_data() {
+    desc 'write initial data, read from all'
+    run_mongo \
+        'db.createUser({user:"myApp",pwd:"myPass",roles:[{db:"myApp",role:"readWrite"}]})' \
+        "userAdmin:userAdmin123456@$cluster.$namespace"
+    sleep 2
+    run_mongo \
+        'use myApp\n db.test.insert({ x: 100500 })' \
+        "myApp:myPass@$cluster.$namespace"
+    sleep 10            # for minikube
+    compare_mongo_cmd "find" "myApp:myPass@$cluster-0.$cluster.$namespace"
+    compare_mongo_cmd "find" "myApp:myPass@$cluster-1.$cluster.$namespace"
+    compare_mongo_cmd "find" "myApp:myPass@$cluster-2.$cluster.$namespace"
+}
+
+run_backup() {
+    local name=$1
+    local idx=$2
+
+    desc "run backup $name-$idx"
+    cat $test_dir/conf/$name.yml \
+        | $sed -e "s/name:/name: $name-$idx/" \
+        | kubectl_bin apply -f -
+    wait_backup "$name-$idx"
+}
+
+check_recovery() {
+    local backup_name=$1
+    local restore_type=$2
+    local restore_date=$3
+    local cmp_postfix=$4
+
+    desc "write more data before restore by $restore_type"
+    sleep 60
+    run_mongo \
+        'use myApp\n db.test.insert({ x: 100501 })' \
+        "myApp:myPass@$cluster.$namespace"
+
+    desc 'waiting for chunks to be uploaded'
+    sleep 545
```

Comment:
is 10 minutes really needed?
I mean we have just few lines of data, but wait for 10 min + 1 min before insert

Reply:
PBM uploads chunks to a bucket with a delay of 10 mins ([docs](https://www.percona.com/doc/percona-backup-mongodb/point-in-time-recovery.html#incremental-backups), it's also ensured by manual tests)

According to docs, delay can be shorter in case of disabling pitr, but is it correct to test in this way? This is probably not what a user is going to do

Reply:
got it